Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
+        puts dylib
+        files.each do |f|
+          unprefixed = f.sub "#{@keg.to_s}/", ""
+          puts "  #{unprefixed}"
```

Comment:
Similarly to the previous comment, I think this needs to be indented by two more spaces if the “Links to:” header shall stay.